data/darktableconfig.xml.in

```diff
@@ -2318,4 +2318,11 @@
     <shortdescription>overwrite the screen's ppd setting</shortdescription>
     <longdescription>if this value is > 0.0 then it is used as the screen's ppd setting which is used for HiDPI support</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/preview/full_size_preload_count</name>
+    <type>int</type>
+    <default>3</default>
```

Comment:
Please correct me if I'm wrong -- while it does have a cost, the job queue is a stack, so this cost is "only" in terms of potentially wasted energy and clogged cache, not in terms of latency. If you move "too quickly", the old jobs won't be executed until all newer requests have been processed. At that time, you're decoding an image that isn't going to be needed anymore.

On the other hand, having a higher number of items for preload means that you have a buffer of stuff you can show immediately, without slowing the user down. That's a big improvement to me.

But anyway, your preference is as good as mine. Number three is just a random pick -- it was my first choice and it's worked well for me. If there's a consensus for changing this, I don't really care, it's configurable after all.